source/async_stack.cpp

```diff
@@ -145,6 +149,20 @@ static thread_local AsyncStackRootHolder currentThreadAsyncStackRoot;
 
 }  // namespace
 
+namespace utils {
+  uint64_t getOSThreadID() {
+   #if defined(__APPLE__)
+     uint64_t tid;
+     pthread_threadid_np(nullptr, &tid);
+     return tid;
```

Comment:
I think this Apple-specific implementation implies there's no need for an Apple-specific definition of `UNIFEX_SYS_gettid`, right?